cms/djangoapps/course_creators/admin.py

```diff
@@ -75,22 +111,31 @@ def save_model(self, request, obj, form, change):
         obj.admin = request.user
         obj.save()
 
+    # This functions is overriden to update the m2m query
+    def save_related(self, request, form, formsets, change):
+        super().save_related(request, form, formsets, change)
+        state = form.instance.state
+        if state != CourseCreator.GRANTED:
+            form.instance.organizations.clear()
```

Comment:
This functionality--specifically, that the many-to-many clears itself if `GRANTED` is not chosen--could be a bit confusing to users of the admin panel. It is not typical that a form silently clears itself based on the contents on another piece of the form. Is there a reason we need to do this?

Reply:
This can be disregarded ^